Sources/ApodiniVaporSupport/HTTPStatus+ErrorType.swift

```diff
@@ -0,0 +1,29 @@
+//
+// Created by Andreas Bauer on 03.07.21.
+//
+
+import Apodini
+import Vapor
+
+extension Vapor.HTTPStatus {
+    /// Creates a `Vapor``HTTPStatus` based on an `Apodini` `ErrorType`.
+    /// - Parameter status: The `Apodini` `ErrorType` that should be transformed in a `Vapor``HTTPStatus`
+    public init(_ error: Apodini.ErrorType) {
```

Comment:
I think it would make sense to make this an `ApodiniErrorCompliantOption`.

Reply:
I think this is might be still a leftover back when this PR contained support for ApodiniError in the REST exporter (before you added it with your PR). If I remember correctly you already introduced a ApodiniErrorCompliantOption for the HTTPStatus.